README.md

```diff
@@ -5,13 +5,12 @@ Cloud environments with [Dask Gateway](https://docs.dask.org/ "Parallel computin
 [![PyPI version](https://badge.fury.io/py/qhub.svg)](https://badge.fury.io/py/qhub)
 
 QHub is an open source tool that enables users to build and maintain
-cost-effective and scalable compute/data science platforms [on
-hpc](#qhub-hpc) or on [kubernetes](#qhub) with minimal DevOps
+cost-effective and scalable compute/data science platforms on [HPC](#qhub-hpc) or on [Kubernetes](#qhub) with minimal DevOps
 experience.
 
-Not sure what to choose? Check out our [FAQ](../02_get_started/07_support.md#faq).
+Not sure what to choose? Check out our [Setup Initialization](docs/source/02_get_started/02_setup.md) page.
 
-**This repository details the [QHub](https://qhub.dev/ "Official QHub docs") version.**
+**This repository details the [QHub](https://qhub.dev/ "Official QHub docs") (Kubernetes) version.**
```

Comment:
I think this should come before the ###Qhub section title (right before the HPC one, is a little strange)

Reply:
You're right! Thanks for spotting 🎖️